examples/halo_test/halo_test.f90

```diff
@@ -618,7 +618,7 @@ subroutine test_halo_size(arrh, nx_expected, ny_expected, nz_expected, tag)
 
          all_pass = .false.
       else
-         write (*, *) trim(rank_lbl), " ", tag, ":PASS"
+         !write (*, *) trim(rank_lbl), " ", tag, ":PASS"
```

Comment:
Keep the line or remove it. If needed, keep it and skip it when the debug mode is not active.

Reply:
I still think it is best to remove the first iteration since it tend to skew the results. After it iteration time tends to be relatively constant and the 50 iterations smooth the ups and down. 
In practice you’ll not see the effect of the first iteration in a simulation, but to take it out we’ll need around 100 iterations which is uncessary and because the large number of tests we are doing  start to make the testing a bit too long.